atlas-content-modeler/tutorial/models/fields/relationship/reverse-relationships.mdx

```diff
@@ -0,0 +1,40 @@
+---
+slug: /atlas-content-modeler/tutorial/models/fields/relationship/reverse_relationships
+title: Atlas Content Modeler Fields - Relationship - Reverse Relationships
+description: Learn how to use Reverse Relationships in Atlas Content Modeler.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../../../introduction.mdx).
+
+## Relationship Field (Beta) - Reverse Relationships
+
+The Reverse Relationship option in Relationship Field is used to display the corresponding linked model in the publisher side of the application for that particular model. It is also used to expose the Reverse Relationship in WPGraphQL.
```

Comment:
```suggestion
The Reverse Relationship option displays the corresponding linked model in the publisher side of the application for that particular model. You can also use Reverse Relationship to expose the Reverse Relationship in WPGraphQL.
```

Reply:
@matthewguywright looks like you missed another suggestion here. See the one below too. Otherwise this looks ready.